mw-webapp/src/dataAccessLogic/getDayReports.ts

```diff
@@ -41,10 +44,18 @@ export const getDayReports = async (): Promise<DayReport[]> => {
       return problemForCurrentPeriod;
     });
 
+  const mentorComments = firstReport.mentorComments.
+    map((mentorCommentUuid) => {
+      const mentorComment: MentorComment = mentorCommentsPreview
+        .find((elem) => elem.uuid === mentorCommentUuid) ?? {} as MentorComment;
```

Comment:
avoid as

Reply:
Don't know how to avoid it, added Error if mentor comment is not found, i dont know if its better approarch